kolibri/plugins/coach/assets/src/views/learners/LearnerSummaryPage/LearnerHeader.vue

```diff
@@ -55,7 +55,10 @@
           </p>
           <div class="value-box">
             <p class="value">{{ lessonsCompleted }}</p>
-            <p style="display: inline; word-wrap: break-word">
+            <p
```

Comment:
`0 of 0` does seem a little unhelpful (and inconsistent with the other displays).